src/utils.py

```diff
@@ -69,3 +85,31 @@ def print_os_environment():
     """
     for key, val in os.environ.items():
         print(f"{key}: {val}")
+
+
+def semver_labels_exist_in_pull_labels(pull):
+    return any([label for label in get_labels(pull=pull) if label in SEMVER_LABELS_SET])
```

Comment:
`any` here is not needed as either there will be labels in the list or it ill be an empty list, how about:
`any([label in SEMVER_LABELS_SET] for label in get_labels(pull=pull))`

?